src/example-types/date.js

```diff
@@ -33,9 +36,15 @@ module.exports = {
     let gte = from
     let lte = to
 
-    let getDateIfValid = x =>
-      moment.utc(new Date(x)).isValid() &&
-      moment.utc(new Date(x)).format('YYYY-MM-DD')
+    let getDateIfValid = x => {
+      let isValid = moment.utc(new Date(x)).isValid()
+      if (isValid) {
+        return context.isoFormat
```

Comment:
instead of exposing the fact that its isoFormat, maybe we should just have a flag like `includeTime`, similar to how es itself does "dateOptionalTime"